client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -38,6 +38,8 @@ export class DowngradeStep extends Component {
 		const canRefund = !! parseFloat( refundAmount );
 		const amount = currencySymbol + ( canRefund ? refundAmount : planCost );
 		const isEnglishLocale = [ 'en', 'en-gb' ].indexOf( userUtils.getLocaleSlug() ) >= 0;
+		const downgradeWarning =
```

Comment:
Sorry, I forgot to mention to you about translation - all new strings need to be translatable - see the other uses of translate() in this file for examples, and https://github.com/Automattic/i18n-calypso/blob/master/README.md give more details about translation options.